src/components/Textarea/Textarea.js

```diff
@@ -73,13 +79,17 @@ Textarea.propTypes = {
     label: PropTypes.string
   }).isRequired,
   variant: PropTypes.oneOf(['default', 'codearea']),
-  size: PropTypes.oneOf(['small', 'medium'])
+  size: PropTypes.oneOf(['tiny', 'small', 'medium']),
+  hasResize: PropTypes.bool,
+  maxHeight: PropTypes.number
 }
 
 Textarea.defaultProps = {
   className: '',
   size: 'medium',
-  variant: 'default'
+  variant: 'default',
+  hasResize: false,
+  maxHeight: null
```

Comment:
Moim zdaniem wywali Ci tutaj bład w propsach, oczekujesz number a podasz nulla :)
Dużo lepiej jeżeli propert nie jest required byłoby zdefiniować defaultPropsa jako undefined, wtedy to jest tak jakbyś tego propsa nie podał do komponentu w ogóle, a tak z defaultu dostaniesz nulla

Reply:
możesz też rozszerzyć oczywiście Propsa o możliwość dostania nulla :P

Reply:
> Moim zdaniem wywali Ci tutaj bład w propsach, oczekujesz number a podasz nulla :) Dużo lepiej jeżeli propert nie jest required byłoby zdefiniować defaultPropsa jako undefined, wtedy to jest tak jakbyś tego propsa nie podał do komponentu w ogóle, a tak z defaultu dostaniesz nulla

Nie wywala i w ogóle nie ustawia styli gdy jest null. Mogę to zmienić. Co myślisz @nataliapietrzak-landingi

Reply:
Myślę, że można zostawić. Ja wolę ustawiać nulle, a undefined zostawiać komputerowi

Reply:
Ok skoro nie wpływa na działanie to nie ma to większego znaczenia :)